app/views/ControllerInputsView.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControllerInputsView
+ * @requires models/Commands
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires config/vehicle-channel-config
+ * @requires config/picpilot-config
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying all the received controller inputs
+ */
+
+var remote = require('electron').remote;
+var Template = require('../util/Template');
+var TelemetryData = remote.require('./app/models/TelemetryData');
+var Commands = remote.require('./app/models/Commands');
+var channel_config = remote.require('./config/vehicle-config');
+var picpilot_conf = remote.require('./config/picpilot-config');
+
+module.exports = function (Marionette) {
+
+  return Marionette.ItemView.extend({
+    template: Template('ControllerInputsView'),
+    className: 'controllerInputsView',
+
+    ui: {
+      front23: '.circle.front.twothree',
+      front14: '.circle.front.onefour',
+      ch1_text: '.value.one',
+      ch2_text: '.value.two',
+      ch3_text: '.value.three',
+      ch4_text: '.value.four',
+      auto_pilot_btn: '.auto-pilot',
+    },
+
+    initialize: function () {
+      this.throttle = 0;
+      this.roll = 0;
+      this.pitch = 0;
+      this.yaw_rudder = 0;
+      this.auto_on_off = true;
+      this.control_off = false;
+      this.is_scaled = false;
+      this.type = picpilot_conf.get('vehicle_type');
+    },
+    
+    rcIsOff: function(data){
```

Comment:
repeated code. It'd be better to just make this function take in an array or 5 parameters and let it check if any of them are -10000